src/FixedDataTable.js

```diff
@@ -631,6 +631,7 @@ var FixedDataTable = createReactClass({
           onColumnResize={this._onColumnResize}
           onColumnReorder={onColumnReorder}
           onColumnReorderMove={this._onColumnReorderMove}
+          showScrollbarY={showScrollbarY}
```

Comment:
where's this value coming from?  I don't see it above here in this method

Reply:
Huh, I thought I'd moved that up already.. but it's from L:641 below. It only appeared to work for me  because of the adjustedWidth stuff from my prior fix. I'll push up a fix shortly.